docs/chat_setup/slack_setup.md

```diff
@@ -0,0 +1,91 @@
+# Slack Setup 
+
+These are the four distinct configuration values you will need to configure in `nautobot_config.py`.
+
+| Configuration Setting        | Mandatory? | Default |
+| ---------------------------- | ---------- | ------- |
+| `enable_slack`               | **Yes**    | False   |
+| `slack_api_token`            | **Yes**    | --      |
+| `slack_signing_secret`       | **Yes**    | --      |
+| `slack_slash_command_prefix` | No         | `"/"`   |
+
+These values will be used in the `nautobot_config.py` file, once we get to the section where we cover server configuration.
+For now, take a mental note that in this section where we are configuring the Slack application, we will need to explicitly note the
+`slack_api_token` and `slack_signing_secret` values when they are presented.
+
+Example config snippet from `nautobot_config.py` for configuring chatops with Slack:
+```python
+# Enable installed plugins. Add the name of each plugin to the list.
+PLUGINS = ["nautobot_chatops"]
+
+PLUGINS_CONFIG = {
+    'nautobot_chatops': {
+        'enable_slack': True,
+        'slack_api_token': '<slack-api-token>',
+        'slack_signing_secret': '<slack-signing-secret>',
+        'slack_slash_command_prefix': '/network2-'
+    }
+}
+```
+> Note: this example uses a configured `slack_slash_command_prefix` value of `/network2-` (for use in Slack as `/network2-nautobot`); if not explicitly configured, this value defaults to `/` (for use as `/nautobot`).
```

Comment:
Let's move to another example rather than just nautobot everywhere, such as one of the chat bots that is in development. Grafana would be a fit here.

Reply:
I removed the note in question here.  I also added a note about specific custom development that allows for things like `/grafana` (line 131).  Also, `/grafana` is still used as an example in link 42.